src/components/pages/annual-report-2018/AnnualReportView.scss

```
@@ -0,0 +1,451 @@
+.annual-report {
+  align-items: center;
+  display: flex;
+  flex: 0 0 auto;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.annual-report h2 {
+  color: #222;
+  font-size: 3.2rem;
+  font-weight: 700;
+  line-height: 1.2;
+  max-width: 800px;
+  @media screen and (max-width: 600px) {
+    font-size: 2.4rem;
+    line-height: 1.2;
+  }
+}
+
+.annual-report .blue {
+  color: #0025e4;
+}
+
+.annual-report--overview {
+  padding-bottom: 150px;
+  width: 100%;
+}
+
+.annual-report .wrapper {
+  flex-direction: column;
+  margin: 0 auto;
+  max-width: 1200px;
+  width: 90%;
+}
+
+.annual-report--year {
+  color: #222;
+  font-size: 12vw;
+  @media screen and (min-width: 1400px) {
+    font-size: 12rem;
+  }
+  @media screen and (max-width: 500px) {
+    font-size: 6rem;
+    line-height: 1.2;
+  }
+}
+
+.annual-report--hero {
+  color: #222;
+  font-size: 6vw;
+  letter-spacing: -2px;
+  line-height: 6.875vw;
+  margin-bottom: 50px;
+  max-width: 620px;
+  width: 90%;
+  @media screen and (min-width: 1400px) {
+    font-size: 6rem;
+    line-height: 6.875rem;
+  }
+  @media screen and (max-width: 500px) {
+    font-size: 3rem;
+    line-height: 1.2;
+  }
+}
+
+.annual-report--text {
+  font-size: 18px;
+  line-height: 26px;
+  max-width: 600px;
+  width: 100%;
+}
+
+.annual-report--growth {
+  background-image: url('./assets/team-growth.png');
+  background-position: bottom left;
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 0;
+  margin-top: 60px;
+  padding-bottom: 45%;
+  width: 100%;
+  @media screen and (max-width: 1000px) {
+    padding-bottom: 60%;
+  }
+  @media screen and (max-width: 500px) {
+    padding-bottom: 80%;
+  }
+}
+
+.annual-report--goals {
+  margin-top: 100px;
+  max-width: 950px;
+
+  ul {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin-top: 60px;
+  }
+}
+
+.annual-report--goals ul {
+  li {
+    align-items: center;
+    border-top: 1px solid #222;
+    display: flex;
+    height: 130px;
+    padding: 0 50px;
+    position: relative;
+    width: 45%;
+    @media screen and (max-width: 600px) {
+      height: 100px;
+      padding: 10px 0 0 20px;
+    }
+  }
+}
+
+.annual-report-goals li {
```

```suggestion
.annual-report--goals li {
```

Found it. There was a typo in this selector. It should be `--goals`, not `-goals`.

---

Fixed this in https://github.com/ShelterTechSF/sheltertech.org/pull/50/commits/f45fc1a29497b9476fff6f505fdcdf7ac64c5367.